src/network/remote/GirderClient.js

```diff
@@ -89,15 +89,16 @@ export function build(config = location, ...extensions) {
         };
 
         eventSource.onerror = (e) => {
-          // Wait 10 seconds if the browser hasn't reconnected then
-          // reinitialize.
+          // Wait 10 seconds if the browser hasn't reconnected then reinitialize.
           setTimeout(() => {
             if (eventSource && eventSource.readyState === EventSource.CLOSED) {
               connectToNotificationStream();
             } else {
+              eventSource.close();
               eventSource = null;
```

Comment:
This line alone makes no sense, if EventSource is OPEN then clear the eventSource with `eventSource = null`? 

Now we additionally close the connection, clear it, and reconnect it. Will need to work with the change to see if it makes a difference.